### DISCUSSION TITLE
HOROVOD_MPI_HOME no longer honored but can't find equivalent feature

### DISCUSSION THREAD

**Discussion by Lana-Deere** [horovod/horovod#3713] [Q&A]
I'm trying to pip install horovod[tensorflow] on Rocky Linux 9 (RedHat 9 clone) with g++ 11.2.1, python 3.9.13, tensorflow 2.10.  Last time I installed horovod there was an environment variable HORVOD_MPI_HOME which permitted me to specify the version of MPI I wanted used.  That variable does not seem to be honored any longer but I could not find an equvalent in the documentation.  How should I specify that I want a particular installation of MPI to be used (which is not the same as the OS-provided version)?

**Comment by maxhgerlach** [comment]
Hi @Lana-Deere,

could you try setting just `MPI_HOME`? That's a CMake-specific environment variable now. (Alternatively adapting `MPI_<LANG>_COMPILER` for `C` and `CXX` to the paths of your `mpicc` and `mpicxx` might work, similarly to https://github.com/horovod/horovod/issues/3696#issuecomment-1248921736)

If either approach works, I agree that we should improve documentation in this regard.

**Reply by Lana-Deere** [reply]
I tried setting MPI_<LANG>_COMPILER but it will be a while before I can be sure that it works, as I will have to port more of the stuff I'm working with to Rocky9 first.
